app/init.php

```diff
@@ -1187,58 +1187,57 @@ function (mixed $value) {
 App::setResource('queueForMigrations', function (Connection $queue) {
     return new Migration($queue);
 }, ['queue']);
-App::setResource('clients', function ($request, $console, $project) {
-    $console->setAttribute('platforms', [ // Always allow current host
-        '$collection' => ID::custom('platforms'),
-        'name' => 'Current Host',
-        'type' => Origin::CLIENT_TYPE_WEB,
-        'hostname' => $request->getHostname(),
-    ], Document::SET_TYPE_APPEND);
-
-    $hostnames = explode(',', System::getEnv('_APP_CONSOLE_HOSTNAMES', ''));
+App::setResource('platforms', function (Document $project, Document $console) {
+    $platforms = [];
+    $platforms[] = $project->getAttribute('platforms', []);
+    $platforms[] = $console->getAttribute('platforms', []);
+    return $platforms;
+}, ['project', 'console']);
+App::setResource('hostnames', function (array $platforms) {
+    // Always allow console hostname
+    $hostnames = [System::getEnv('_APP_CONSOLE_HOSTNAME', 'console')];
```

Comment:
I believe this should be `$hostnames = ['console'];` ? 
Because we get and split the `_APP_CONSOLE_HOSTNAME` in the line below